### PR TITLE
Rmi test

### PR DESCRIPTION
**Pull request by SamanthaDawsonBanks**

### CODE REVIEW

**Comment by SamanthaDawsonBanks** [comment]
there are still some debug and testing code bits in but should mostly work

File: mcomp_C/mcomp_C.ino [hunk 1]
```diff
@@ -77,22 +52,7 @@ void setup() {
...
-  //base64 decode
-  //eval read to command/data
-  //switch (sense)
-  //switch (drive-data)
-
-  //if data waiting
-  //encode 64
-  //serial write
-
+  //DEBUG.println("Moo");
```
Comment (RyanShoobert) [inline mcomp_C/mcomp_C.ino:55]: Moo?

File: mcomp_C/movement/Propulsion.cpp [hunk 2]
```diff
@@ -64,12 +63,15 @@ Waypoint Propulsion::Drive (Waypoint w){
    * This has now been factored out to datatype AngleDistance
    */
   AngleDistance movement = w.toAngleDistance();
-  currentHeading += rotate(movement.getTheta()-currentHeading);
+  currentHeading = currentHeading
+      + rotate(movement.getTheta() - currentHeading);
+  delay(100);
+  delay(1000);
```
Comment (RyanShoobert) [inline mcomp_C/movement/Propulsion.cpp:69]: Is there for a need for both delays?

**Review by RyanShoobert — APPROVED** [review]

File: mcomp_C/movement/Propulsion.cpp (hunk 2 above)
Comment (SamanthaDawsonBanks) [reply mcomp_C/movement/Propulsion.cpp:69]: one (short) is for motor spin down second (long) is for visual debugging between rotate and drive

File: mcomp_C/mcomp_C.ino (hunk 1 above)
Comment (SamanthaDawsonBanks) [reply mcomp_C/mcomp_C.ino:55]: lol debug comments

**Review by stevempope — APPROVED** [review]

**Pull request merged**